src/gosling-track/gosling-track.ts

```diff
@@ -1014,52 +1061,182 @@ function GoslingTrack(HGC: any, ...args: any[]): any {
             return mergedCapturedElements;
         }
 
-        onMouseOut() {
-            document.body.style.cursor = 'default';
-            this.mouseOverGraphics.clear();
+        /**
+         * Highlight marks that are either hovered or selected.
+         */
+        highlightMarks(
+            g: Graphics,
+            marks: MouseEventData[],
+            stroke: string,
+            strokeWidth: number,
+            strokeOpacity: number,
+            color: string,
+            fillOpacity: number
+        ) {
```

Comment:
In case there end up being more options, I'd change this signature to:

```typescript

        highlightMarks(
            g: Graphics,
            marks: MouseEventData[],
            options: {
                stroke: string,
                strokeWidth: number,
                strokeOpacity: number,
                color: string,
                fillOpacity: number
            }
        ) {
```